.changeset/red-otters-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+fix rootRelationalClassMapping compilation to include filter
```

Comment:
Let's start with a capitalized word and end with a period for consistency

Reply:
Thanks done.